services/director-v2/src/simcore_service_director_v2/api/routes/computations.py

```diff
@@ -92,13 +92,13 @@ async def create_computation(
         if is_pipeline_running(pipeline_state):
             raise HTTPException(
                 status_code=status.HTTP_403_FORBIDDEN,
-                detail=f"Projet {job.project_id} already started, current state is {pipeline_state}",
+                detail=f"Projet {job.project_id=} already started, current {pipeline_state=}",
```

Comment:
one question here. you print variable names... this goes out in the public... I am not sure it makes any sense to have the variable names there...